openupgrade_scripts/scripts/payment_transfer/15.0.2.0/post-migration.py

```diff
@@ -4,4 +4,10 @@
 @openupgrade.migrate()
 def migrate(env, version):
     openupgrade.load_data(env.cr, "payment_transfer", "15.0.2.0/noupdate_changes.xml")
+    # Delete the old form record and link manually link the custom acquirers made by
+    # the users.
     openupgrade.delete_records_safely_by_xml_id(env, ["payment_transfer.transfer_form"])
+    custom_transfer = env["payment.acquirer"].search(
+        [("provider", "=", "transfer"), ("redirect_form_view_id", "=", False)]
```

Comment:
Shouldn't they be all of them?

Reply:
Every acquirer type could have a different acquire redirect form

Reply:
I mean all the transfer one, although they have defined `redirect_form_view_id`. It will probably be incorrect.

Reply:
Ok, done